### PR TITLE
feat(global-writes): open global writes tab when creating a collection COMPASS-8280

### PR DESCRIPTION
**Pull request by mabaasit**
### Checklist
- [ ] New tests and/or benchmarks are included
- [ ] Documentation is changed or added
- [ ] I have signed the MongoDB Contributor License Agreement (https://www.mongodb.com/legal/contributor-agreement)

## Motivation and Context

- [ ] Bugfix
- [ ] New feature
- [ ] Dependency update
- [ ] Misc

## Types of changes

- [ ] *Backport Needed*
- [ ] Patch (non-breaking change which fixes an issue)
- [ ] Minor (non-breaking change which adds functionality)
- [ ] Major (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: packages/databases-collections/src/modules/create-namespace.ts [hunk 1]
```diff
@@ -402,17 +403,20 @@ export const createNamespace = (
...
+
+      track(`${kind} Created`, trackEvent, connectionInfo);
 
       globalAppRegistry.emit('collection-created', namespace, {
         connectionId,
       });
       workspaces.openCollectionWorkspace(connectionId, namespace, {
         newTab: true,
+        initialSubtab:
+          connectionInfo && connectionSupports(connectionInfo, 'globalWrites')
```
Comment (paula-stacho) [inline packages/databases-collections/src/modules/create-namespace.ts:417]: connectionSupports won't be sufficient. we'll need to be filtering out the admin/config/local collections and also views. please sync with @djechlin-mongodb
more context: https://mongodb.slack.com/archives/C03DU5ATTB6/p1729869659313749

Comment (paula-stacho) [reply packages/databases-collections/src/modules/create-namespace.ts:417]: sorry for only realising this now 🙈

Comment (gribnoysup) [reply packages/databases-collections/src/modules/create-namespace.ts:417]: You can't create views through this modal and I'm pretty sure that even db users with admin roles can't create collections inside special-ish namespaces (but for anything that is defined as special-ish by mongodb-ns package we probably can add a filter here, yeah)

Comment (paula-stacho) [reply packages/databases-collections/src/modules/create-namespace.ts:417]: ah right, thanks @gribnoysup !

Comment (mabaasit) [reply packages/databases-collections/src/modules/create-namespace.ts]: added in [0d124af](https://github.com/mongodb-js/compass/pull/6405/commits/0d124af0cf4fe8d7b81d29e8ef30c2b48bccbdbf)

**Review by paula-stacho — APPROVED** [review]
lgtm

**Pull request merged**